articles/openshift/support-policies-v4.md

```diff
@@ -10,26 +10,63 @@ ms.date: 03/05/2021
+* Don't remove or modify the worker or master machine config pools.
+* Don't pause any machineconfigpool.
+
+### Cluster Operators
 
 * All OpenShift Cluster operators must remain in a managed state. The list of cluster operators can be returned by running `oc get clusteroperators`.
-* The cluster must have a minimum of three worker nodes and three manager nodes. Do not have taints that prevent OpenShift components to be scheduled. Do not scale the cluster workers to zero, or attempt a graceful cluster shutdown.
+* Following configuration of cluster operators such as authentication or dns, the associated cluster operator should remain in a available state without any errors.  After applying configuration ensure that `oc get clusteroperators` shows all cluster operators as healthy.
+
+### OpenShift Resources
+
+* You may not remove or replace any native Azure Red Hat OpenShift components or any other component installed and managed by Azure Red Hat OpenShift
 * Don't remove or modify the cluster Prometheus and Alertmanager services.
 * Don't remove Service Alertmanager rules.
-* Don't remove or modify network security groups.
-* Don't remove or modify Azure Red Hat OpenShift service logging (mdsd pods).
-* Don't remove or modify the 'arosvc.azurecr.io' cluster pull secret.
+* Don't remove or modify any resources in any namespace beginning with `openshift-azure`
+* Don't run workloads in the kube-system namespace or any namespace beginning with "openshift-".
+* Don't remove or modify the 'arosvc.azurecr.io' cluster pull secret.  This secret can be seen with `oc -n openshift-config get secret pull-secret`.
+* Customer workloads must not be scheduled on the master nodes to prevent any resource contention with the OpenShift master components.
+* Don't set any unsupportedConfigOverrides options. Setting these options prevents minor version upgrades.
+* A pod disruption budget should not set a `maxUnavailable` of 0% or 0, or a `minAvailable` of 100% or equal to the number of replicas.  This can block nodes from being drained and upgraded.
+* Don't create or enable any Feature Gates
+* Don't create or enable the cluster-wide proxy
+* Don't modify the audit log policy
+* Don't modify or remove the OpenShift SDN network or attempt to use another SDN provider.
+* Don't modify the cluster network OpenShift SDN configuration.
+* Azure Red Hat OpenShift does not support adding an additional network such as a bridge, host-device, ipvlan, or macvlan network.
+* The cluster version resource should only use stable release channels for upgrades.  There is no support for non-stable channels.  The channel can be seen by running `oc get clusterversion version -o jsonpath="{.spec.channel}"`.  In new cluster installations, the channel may not be set yet.
+* Any workloads that are run on the cluster including operators from OperatorHub must not affect running OpenShift components and cluster operators from being available.
```

Comment:
May also be worth adding a line about performing admin tasks via SSH, including etcd recoveries and certificate recoveries (both of which have step by step guides in the OpenShift docs) and attempted upgrade rollbacks (not documented, but attempted pretty regularly)